### DISCUSSION TITLE
External links in headers

### DISCUSSION THREAD

**Discussion by lazarusA** [vuejs/vitepress#3731] [Q&A] [answered]
Is there a setting so that the following renders correctly ?

```md
### [wiki](https://www.wikipedia.org) (Wikipidia example)
```
currently, is renderer like this:

[wiki](https://www.wikipedia.org/) (Wikipidia example) {#[wiki](https://www.wikipedia.org/)-(Wikipidia-example)}

and the expected output is just

[wiki](https://www.wikipedia.org/) (Wikipidia example)

**Comment by brc-dd** [comment, marked as answer]
Seems to generate correct output:

<img width="346" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/40b84d0b-41ec-4579-982a-bb92b94bb4c8">

<img width="384" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/864773e3-ebf2-4ba6-8e74-20b199397d29">
